tests/test_chromatic.py

```diff
@@ -64,6 +66,27 @@
 disk_SED = galsim.SED(os.path.join(datapath, 'CWW_Sbc_ext.sed'), wave_type='ang')
 disk_SED = disk_SED.withFluxDensity(target_flux_density=0.3, wavelength=500.0)
 
+# define the directory containing some reference images
+refdir = os.path.join(".", "chromatic_reference_images") # Directory containing the reference
+
+# Define an ChromaticObject subclass for which interpolation could be helpful, but calculations can
+# be done quickly:
+# It's a Gaussian with a wavelength-dependent sigma [sigma = sigma_0 (wave / 500 nm)]
+# and shear g1 [g1 = 0.1 ((wave)/(500 nm) - 1)]
+class ChromaticGaussian(galsim.ChromaticObject):
```

Comment:
This would be better defined within test_interpolated_ChromaticObject where it is used, rather than here at the top of the file.